src/uk/zenko/src/eu/kanade/tachiyomi/extension/uk/zenko/Zenko.kt

```diff
@@ -0,0 +1,193 @@
+        return makeZenkoMangaRequest(offset, "viewsCount")
+    }
+
+    override fun popularMangaParse(response: Response) = parseAsMangaResponseDto(response)
+
+    // =============================== Latest ===============================
+    override fun latestUpdatesRequest(page: Int): Request {
+        val offset = offsetCounter(page)
+        return makeZenkoMangaRequest(offset, "lastChapterCreatedAt")
+    }
+
+    override fun latestUpdatesParse(response: Response) = parseAsMangaResponseDto(response)
+
+    // =============================== Search ===============================
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        if (query.length >= 2) {
+            val offset = offsetCounter(page)
+            val url = "$API_URL/titles".toHttpUrl().newBuilder()
+                .addQueryParameter("limit", "15")
+                .addQueryParameter("offset", "$offset")
+                .addQueryParameter("name", query)
+                .build()
+            return GET(url, headers)
+        } else {
+            throw UnsupportedOperationException("Запит має містити щонайменше 2 символи / The query must contain at least 2 characters")
+        }
+    }
+
+    override fun searchMangaParse(response: Response) = parseAsMangaResponseDto(response)
+
+    // =========================== Manga Details ============================
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        val mangaId = manga.url.substringAfterLast('/')
+        val url = "$API_URL/titles/$mangaId"
+        return GET(url, headers)
+    }
+
+    override fun mangaDetailsParse(response: Response) = SManga.create().apply {
+        val mangaDto = response.parseAs<MangaDetailsResponse>()
+        title = mangaDto.engName ?: mangaDto.name
+        thumbnail_url =
+            "$IMAGE_STORAGE_URL/${mangaDto.coverImg}?optimizer=image&width=560&quality=70&height=auto"
+        url = "/titles/${mangaDto.id}"
+        description = "${mangaDto.name}\n${mangaDto.description}"
+        genre = mangaDto.genres!!.joinToString { it.name }
+        author = mangaDto.author!!.username
+        status = mangaDto.status.toStatus()
+    }
+
+    // ============================== Chapters ==============================
+    override fun chapterListRequest(manga: SManga): Request {
+        val mangaId = manga.url.substringAfterLast('/')
+        val url = "$API_URL/titles/$mangaId/chapters"
+        return GET(url, headers)
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val result = response.parseAs<List<ChapterResponseItem>>()
+        return result.sortedByDescending { item ->
+            val id = StringProcessor.generateId(item.name)
+            if (id > 0) id else item.id.toDouble()
+        }.map { chapterResponseItem ->
+            SChapter.create().apply {
+                url = "/titles/${chapterResponseItem.titleId}/${chapterResponseItem.id}"
```

Comment:
Use `setUrlWithoutDomain()` so responsibility is left to the same piece of code, for consistency.